bigquery/deploy/README.md

```diff
@@ -47,6 +47,8 @@
     * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
     * Change name to `elasticsearch-cluster`
     * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
+    If you need more memory for Elasticsearch, you may need `n1-highmem-4`. [Elasticsearch recommends](https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html) the VM has at least
```

Comment:
The change to add `es_java_opts` to deploy.json and dynamically change es-data-stateful.yaml will be in a different PR? That's fine, just wondering.

Reply:
Sorry, added to this PR.